src/data/faq_de.json

```diff
@@ -131,6 +131,17 @@
                             "</ul>"
                         ]
                     },
+                    {
+                        "title": "Warum bekomme ich die Fehlermeldung 'Dieser QR-Code ist kein gültiges Impfzertifikat (VC_PREFIX_INVALID)', wenn ich versuche meinen Impf-QR-Code zu scannen?",
+                        "anchor": "vac_cert_index",
+                        "active": true,
+                        "textblock": [
+                            "Es gibt derzeit (Stand: 9 Juni 2021) noch keine Impfzertifikate mit QR-Codes, die mit der App eingescannt werden können.",
+                            "Das in der Vergangenheit und momentan ausgestellte Dokument ist trotz des QR-Codes kein gültiges digitales Impfzertifikat, sondern lediglich eine Impfbescheinigung, die dem deutschen Infektionsschutzgesetz folgt, nicht aber den aktuellen neuen abgestimmten europäischen Richtlinien für ein digitales Impfzertifikat.",
+                            "Diese neuen digitalen Impfzertifikate werden demnächst ausgestellt. Solche digitalen Zertifikate können nur von autorisierten Stellen über die zentrale Schnittstelle bereitgestellt werden, die im Auftrag der Bunderegierung von anderen Partnern entwickelt wurde. Der Feldtest dafür wurde inzwischen beendet, und die Funktion wird nun flächendeckend ausgerollt.",
+                            "Es ist noch nicht abschließend geklärt, wer genau neben den Impfzentren auch in der Lage sein wird, ein Impfzertifikat auf Basis eines Impfnachweises, d.h. dem gelben Impfbuch bzw. der oben genannten Impfbescheinigung, auszustellen. Das Netz wird stufenweise weiter aufgebaut."

```

Comment:
Impfausweis instead of Impfbuch.

Reply:
ok, thanks.

Reply:
My yellow book has the title "INTERNATIONALE BESCHEINIGUNGEN ÜBER IMPFUNGEN UND IMPFBUCH" on the front of it, so I think "Impfbuch" is OK, or is there some reason to use "Impfausweis" instead?